loopchain/blockchain/epoch.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, block_manager, leader_id=None):
         self.__complain_vote = dict()  # complain vote dict { round : Vote }
         self.__complain_vote[self.round] = Vote(Epoch.COMPLAIN_VOTE_HASH, ObjectManager().channel_service.peer_manager)
         self.complained_result = None
+        self.complain_duration = conf.TIMEOUT_FOR_LEADER_COMPLAIN
```

Comment:
How about calculating each time instead of storing value?

```python
@property
def complain_duration(self):
    return (2 ** self.round) * conf.TIMEOUT_FOR_LEADER_COMPLAIN
```